packages/react-components/src/components/RichTextEditor/Plugins/PlaceholderPlugin.tsx

```diff
@@ -9,6 +9,7 @@ import type { IEditor } from 'roosterjs-content-model-types';
 export class PlaceholderPlugin extends WatermarkPlugin {
   private isPlaceholderShown: boolean = false;
   private editorValue: IEditor | null = null;
+  private disposer: (() => void) | null = null;
```

Comment:
Again, curious why null specifically needs to be used here, rather than undefined?